jar-service/src/main/java/com/sixsq/slipstream/api/APIBaseResource.java

```diff
@@ -73,10 +77,18 @@ private String getAPIResourceAsJSON() throws IOException {
     }
 
     private String getNameRoles() {
-        String userName = getUser().getName();
-        String roleAdminIfSuper = getUser().isSuper() ? " ADMIN" : "";
 
-        return userName + roleAdminIfSuper;
+        List<String> nameRoles = new ArrayList<>();
+
+        nameRoles.add(getUser().getName());
+        if (getUser().isSuper()) {
+            nameRoles.add("ADMIN");
+        }
+        if (getUser().getRoles() != null) {
+            nameRoles.addAll(Arrays.asList(StringUtils.split(getUser().getRoles(), ",")));
```

Comment:
This should strip out any special roles that we have.  From memory these are "ADMIN", "USER", and "ANON".